### PR TITLE
add initial Greek translation pages

### PR DESCRIPTION
**Pull request by Kosmas**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -17,10 +17,11 @@ description:
   vi: Giáo trình cho ngôn ngữ lập trình Elixir
   ru: Уроки программирования на языке Elixir
   sk: Lekcie o programovacom jazyku Elixir
-  fr: Leçons sur le langage de programmation Elixir
+  gr: Μαθήματα προγραμματισμού της γλώσσας Elixir
+  fr: Lessons about the Elixir programming language
```
Comment (doomspork) [inline _config.yml]: Hi @Kosmas, could you correct this change?  Looks like the French translation is getting removed.

File: _config.yml [hunk 2]
```diff
@@ -34,7 +35,8 @@ sections:
       vi: Cơ bản
       ru: Основы
       sk: Základy
-      fr: Bases
+      gr: Βασικά
+      fr: Basics
```
Comment (doomspork) [inline _config.yml]: This one and the other French translations appear to be back to English.

**Comment by Kosmas** [comment]
I'm sorry @doomspork. Not sure how it happened as I recreated my fork and the changes. Should be corrected now :-(

**Comment by pjhampton** [comment]
Awesome! Thanks so much @Kosmas. Can you please squash these commits into 1 and I or @doomspork  will get this merged today at some stage?

**Comment by Kosmas** [comment]
#219

**Pull request closed without merging**